scoresheets/SticklerRules.tex

```diff
@@ -0,0 +1,12 @@
+
+The maximum time for this test is 10 minutes.
+
+\begin{scorelist}
+	\scoreheading{Main Goal}
+	\scoreitem[5]{100}{Recognize guests breaking rules and ask them to stop}
```

Comment:
Same as in the test file, my suggestion for the scoresheet is:

*Identify and detain all guests breaking the [house] rules*